releasenotes/notes/0.19/full_prec_sympy-aeee8210091ef20f.yaml

```diff
@@ -0,0 +1,14 @@
+---
+other:
+  - |
+    The string cast for :class:`qiskit.circuit.ParameterExpression` does not
+    have full precision anymore. This removes the trailing 0s when printing
+    parameters that are bound to floats.  This has consequences for QASM
+    serialization and the circuit text drawer::
+
+    .. code-block:: python
+
+      >>> from qiskit.circuit import Parameter
+      >>> x = Parameter('x')
+      >>> str(x.bind({x:0.5}))
+      '0.5'   # instead of '0.500000000000000'
```

Comment:
<img width="734" alt="Screenshot 2021-12-04 at 08 22 49" src="https://user-images.githubusercontent.com/766693/144701466-95ee9998-fd00-4925-840c-4d31b4a81c8a.png">



```suggestion
    .. code-block:: python

        >>> from qiskit.circuit import Parameter
        >>> x = Parameter('x')
        >>> str(x.bind({x:0.5}))
        '0.5'   # instead of '0.500000000000000'
```

Reply:
Done in 1423ebf.